Tests fail during linking due to multiple definition error

Linking fails due to `GetStaticSchedule` not being inline.

`ninja check-clad-firstderivative`

produces


```
FAILED: unittests/Basic/DifferentiatorHTests 
: && /usr/lib/llvm-20/bin/clang -fopenmp -I/usr/lib/llvm-20/lib/clang/20/include/ -I/usr/include/ -fopenmp -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wmisleading-indentation -Wctad-maybe-unsupported -fdiagnostics-color -ffunction-sections -fdata-sections -Wl,--gc-sections unittests/Basic/CMakeFiles/DifferentiatorHTests.dir/DifferentiatorH.cpp.o unittests/Basic/CMakeFiles/DifferentiatorHTests.dir/MultiIncludeDifferentiatorH.cpp.o -o unittests/Basic/DifferentiatorHTests  /usr/lib/llvm-20/lib/libLLVM.so.1  -lstdc++  -lpthread  -lm  unittests/googletest-prefix/src/googletest-build/lib//libgtest.a  unittests/googletest-prefix/src/googletest-build/lib//libgtest_main.a  unittests/googletest-prefix/src/googletest-build/lib//libgmock.a  unittests/googletest-prefix/src/googletest-build/lib//libgmock_main.a && :
/usr/bin/ld: unittests/Basic/CMakeFiles/DifferentiatorHTests.dir/MultiIncludeDifferentiatorH.cpp.o: in function `clad::GetStaticSchedule(int, int, int, int*, int*)':
MultiIncludeDifferentiatorH.cpp:(.text._ZN4clad17GetStaticScheduleEiiiPiS0_+0x0): multiple definition of `clad::GetStaticSchedule(int, int, int, int*, int*)'; unittests/Basic/CMakeFiles/DifferentiatorHTests.dir/DifferentiatorH.cpp.o:DifferentiatorH.cpp:(.text._ZN4clad17GetStaticScheduleEiiiPiS0_+0x0): first defined here
```

--- a/include/clad/Differentiator/Differentiator.h
+++ b/include/clad/Differentiator/Differentiator.h
@@ -821,8 +821,8 @@ T& back(
   template <unsigned N> struct EnzymeGradient { double d_arr[N]; };
 
 #ifdef _OPENMP
-  void GetStaticSchedule(int lo, int hi, int stride, int* threadlo,
-                         int* threadhi) {
+  inline void GetStaticSchedule(int lo, int hi, int stride, int* threadlo,
+                                int* threadhi) {
     /* Static OpenMP scheduler, identical to what LLVM would use. Each thread
        gets one chunk of consecutive iterations. The number of iterations per
        chunk is aproximately trip_count/num_threads. If the trip count can not

